### ISSUE TITLE
1.18.2 Stacktrace showing up in console on wrong command

### ISSUE THREAD

**Issue by Raidobw2** [Seniorendi/AdvancedPeripherals#339]
### Descripe

If you type a wrong command in the server console and press Enter, this shows up:
```
[01:04:19] [Server thread/ERROR] [ne.mi.ev.EventBus/EVENTBUS]: Exception caught during firing event: Index 0 out of bounds for length 0
        Index: 3
        Listeners:
                0: HIGH
                1: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/CommandEvent;)V
                2: NORMAL
                3: ASM: class de.srendi.advancedperipherals.common.events.Events onCommand(Lnet/minecraftforge/event/CommandEvent;)V
java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
        at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
        at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
        at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
        at java.base/java.util.Objects.checkIndex(Objects.java:359)
        at java.base/java.util.ArrayList.get(ArrayList.java:427)
        at TRANSFORMER/advancedperipherals@0.7.18r/de.srendi.advancedperipherals.common.events.Events.getCommandName(Events.java:73)
        at TRANSFORMER/advancedperipherals@0.7.18r/de.srendi.advancedperipherals.common.events.Events.onCommand(Events.java:54)
        at net.minecraftforge.eventbus.ASMEventHandler_289_Events_onCommand_CommandEvent.invoke(.dynamic)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.EventBus.post(EventBus.java:302)
        at MC-BOOTSTRAP/eventbus@5.0.3/net.minecraftforge.eventbus.EventBus.post(EventBus.java:283)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.commands.Commands.m_82117_(Commands.java:226)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.dedicated.DedicatedServer.m_139665_(DedicatedServer.java:341)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.dedicated.DedicatedServer.m_5703_(DedicatedServer.java:327)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:842)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:685)
        at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.MinecraftServer.m_177918_(MinecraftServer.java:261)
        at java.base/java.lang.Thread.run(Thread.java:833)
```
Versions
Minecraft 1.18.2
Forge 40.1.76
Architectury: architectury-4.9.83-forge.jar
Advanced Peripherals: AdvancedPeripherals-0.7.18r.jar
Modpack: FTB Plexiglass Mountain v1.2.0 in-dev

Unsure if architectury plays a role in this, but I've included its version in case

### Steps to reproduce

1. Install Advanced Peripherals and Architectury
2. Write "a" in server console and press Enter
3. Text shows up

### Multiplayer?

Yes

### Version

1.18.2-0.7.18r

### Minecraft, Forge and maybe other related mods versions

Forge 40.1.76 Minecraft 1.18.2

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/17010632/188551616-ed178415-f931-4d9a-8974-9de1fe085d1e.png)

### Crashlog/log

See description

**Comment by Raidobw2** [comment]
Note: the server does not crash when that happens

**Comment by SirEndii** [comment]
Reopen until release

**Comment by SirEndii** [comment]
Fixed in 0.7.19r

**Comment by Raidobw2** [comment]
Thank you!
